extend-expect.d.ts

```diff
@@ -25,5 +25,6 @@ declare namespace jest {
     ): R
     toHaveValue(value?: string | string[] | number): R
     toBeChecked(): R
+    toBeEmpty(): R
```

Comment:
Shouldn't this be `toBeSelected` instead?

```suggestion
    toBeSelected(): R
```